src/runtime/runtime.cs

```diff
@@ -2185,6 +2276,539 @@ internal static IntPtr GetBuiltins()
         {
             return PyImport_Import(PyIdentifier.builtins);
         }
+
+        private static class Delegates
```

Comment:
Please make `Runtime` a `partial` class and move this into a separate file.

Reply:
I can do this just before pushing. Otherwise it would require me to update the roslyn package, that does automatic conversion of [DllImport] into the pattern with `Delegates`.